src/middlewared/middlewared/plugins/consulkv.py

```diff
@@ -70,6 +71,26 @@ class ConsulService(Service):
             logger.error('===> Consul delete_kv error: %s' % (err))
             return False
 
+    @accepts(Str('region'))
+    async def aws_region(self, region):
+        """
+        Create an aws config file with region.
+
+        Returns:
+                    None
+        """
+        return self._aws_config_file(region, None, None)
+
+    @accepts(Str('key_id'), Str('access_key'))
+    async def aws_credentials(self, key_id, access_key):
+        """
+        Create an aws config file with key_id and access_key.
+
+        Returns:
+                    None
+        """
+        return self._aws_config_file(None, key_id, access_key)
+
```

Comment:
I don't see these methods being used anywhere. What are they for?
Whats the use case, for they existing and being public?